server/routes/users.js

```diff
@@ -3,19 +3,18 @@ const router = express.Router()
 
 const { User } = require( '../database' )
 
-router.post( '/sign-up', (request, response) => {
+router.post( '/sign-up', ( request, response ) => {
   const { username, email, password, lat, long } = request.body
 
-  // TODO: If user exists, return
   User.signUp({ username, email, password, lat, long })
-    .then( user => response.status(201).json( user ))
-    .catch( error => response.status(500).json({ error: error.message  }))
+    .then( user => response.status( 201 ).json( user ))
```

Comment:
How come you're not sending `user` in an object?

Reply:
`user` is already an object/json!